spec.md

```diff
@@ -810,6 +844,12 @@ message CreateVolumeRequest {
   // VOLUME_ACCESSIBILITY_CONSTRAINTS plugin capability, the SP MAY
   // choose where the provisioned volume is accessible from.
   TopologyRequirement accessibility_requirements = 7;
+
+  // The IDs of the volume groups where the volume will be added to.
+  // Note that it is possible for a volume to be placed in one or more
+  // groups on the storage backend even if this field is not specified.
+  // This field is OPTIONAL.
+  repeated string volume_group_id = 8 [(alpha_field) = true];
```

Comment:
I forgot to remove this.  Let me fix it.

Reply:
> This is the place where we have the intersection. I added this for the optional field group_snapshot_id in message Snapshot which is in the ControllerService. Can the group_snapshot_id field in the ControllerService be dependent on a capability in GroupControllerService?

That seems *slightly* cleaner than a capability related to groups defined in the vanilla ControllerService. wonder if anyone else has thoughts on this?

Reply:
I removed capability from the ControllerService.